prometheus/map.jinja

```diff
@@ -35,7 +35,13 @@
 {%-   if args|length > 0 %}
 {%-     for k,v in args -%}
 {%-       if not k or not v %}{% continue %}{% endif -%}
+{%-         if v == True -%}
```

Comment:
Could `v` be "true"?

Reply:
Yes, when I want to set `--collector.ntp`. There are other ways to do this, but by setting `collector.ntp: true` is semantically equivalent.